superbench/benchmarks/micro_benchmarks/sharding_matmul.py

```diff
@@ -0,0 +1,239 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""Module of the ShardingMatmul benchmarks."""
+
+import os
+import time
+
+import torch
```

Comment:
One question: If our microbenchmark depends on PyTorch. Do we need to check it? Because we don't run PyTorch if we don't enable it but we don't check in microbenchmark.

Reply:
Have added TODO.